docs/src/pages/index.mdx

```diff
@@ -14,3 +14,33 @@ This is `inlineCode`.
 ```
 Code block vanilla for now.
 ```
+
+```typescript
+import React from 'react'
+
+// comment here
+
+const Example: React.FC = () => {
+  return (
+    <div>
+      <p>hello world</p>
+    </div>
+  )
+}
+
+export default Example
```

Comment:
Would be cool if there was a way to avoid the export.

Reply:
agreed. i'll need to look into it more